db/migrate/20220511151213_add_type_enum_to_shows.rb

```diff
@@ -0,0 +1,8 @@
+class AddTypeEnumToShows < ActiveRecord::Migration[7.0]
+  def change
+    if column_exists? :shows, :show_type
```

Comment:
Since this is a little more too complex to do and this is not a good practice because you will lose all the `show_type` that was assigned to existing data(rather it will all be defaulted to 0), you should remove this 3 migrations and just modify the initial `create_table` migration. Since you cant rerun existing migrations, you should just do `rails db:drop db:create db:migrate` to recreate the schema and fill the data with the rake task again.